handl/js/templates/delivery-form.html

```diff
@@ -1 +1,32 @@
-this is a form
+<form id='delivery-form' >
+  Name:<input id="collector-name-input" type="text" ng-model="delivery.sender_name"/>
```

Comment:
consider using `element(by.model('delivery.sender_name'))` for testing, just to clear up the code